src/app/login/login.component.ts

```diff
@@ -21,26 +21,21 @@ export class LoginComponent implements OnInit {
 
   authToken: any;
 
-  dialogConfig: MatDialogConfig;
-
   constructor(private userService: UserService,
               private dialogService: DialogService,
               private router: Router) {
   }
 
   ngOnInit() {
-    this.dialogConfig = new MatDialogConfig();
   }
 
   async loginUser() {
     const fetchedUsername = this.username.value;
     const fetchedPassword = this.password.value;
 
     if (fetchedUsername === '' && fetchedPassword === '') {
```

Comment:
Not in this pull request, but (fetchedUsername && fetchedPassword) might work the same

Reply:
Alright, sounds like a good idea, but I will add this as a TODO if it's okay. Don't have too much time :D